docs/paypal.md

```diff
@@ -0,0 +1,169 @@
+# Pay with PayPal
+
+Accept PayPal payments in your Android app using the PayPal Payments SDK.
+
+## How it works
+
+This diagram shows how your client, your server, and PayPal interact:
+
+// TODO - Get a diagram of the payment flow similar to [this](https://developer.paypal.com/braintree/docs/start/overview#how-it-works)
+
+## Eligibility
+
+PayPal is available as a payment method to merchants in multiple [countries and currencies](https://developer.paypal.com/docs/checkout/payment-methods/).
+
+This SDK supports a minimum Android API of 23 or higher.
+Android apps can be written in Kotlin or Java 8 or higher.
+
+## How to integrate
+
+- [Custom Integration](#technical-steps---custom-integration)
+
+### Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID and generate an access token. 
+Follow the steps in [Enable the SDK](https://developer.paypal.com/sdk/in-app/android/#link-enablethesdk) to set up your account and application for PayPal payments.
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used in place of a server SDK.
+
+### Technical steps - custom integration
+
+#### 1. Add the Payments SDK  to your app
+
+In your `build.gradle` file, add the following dependency:
+
+```groovy
+dependencies {
+   implementation "com.paypal.android:paypal-checkout:1.0.0"
+}
+```
+
+Add the following maven credentials to your `build.gradle`:
```

Comment:
This is required to use Native Checkout, but I'm not sure how we can explain to merchants in the docs why this is needed. This is a pretty big pain point in my opinion.

Reply:
Completely agree with you here as well. We can ask the 1P NXO team to refactor 3DS to only include the cardinal SDK for the 1P variant, although I'm not sure how soon this would be picked up 🙁